ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/InteractiveStatementExecutorTest.java

```diff
@@ -98,23 +88,27 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.rules.RuleChain;
+import org.mockito.ArgumentCaptor;
+import org.mockito.ArgumentMatcher;
+import org.mockito.InOrder;
+import org.mockito.Mock;
 import org.mockito.Mockito;
 
-public class InteractiveStatementExecutorTest extends EasyMockSupport {
+public class InteractiveStatementExecutorTest {
 
   private static final Map<String, String> PRE_VERSION_5_NULL_ORIGINAL_PROPS = null;
 
   private KsqlEngine ksqlEngine;
   private InteractiveStatementExecutor statementExecutor;
   private KsqlConfig ksqlConfig;
 
-  private final StatementParser mockParser = niceMock(StatementParser.class);
+  private final StatementParser mockParser = mock(StatementParser.class);
```

Comment:
nit: if we're following the pattern this should `@RunWith(MockitoJUnitRunner.class)` and these should use the `@Mock` annotation

Reply:
Added